graf3d/g3d/src/THelix.cxx

```diff
@@ -293,30 +263,6 @@ void THelix::Print(Option_t *option) const
 }
 
 
-////////////////////////////////////////////////////////////////////////////////
-/// Save primitive as a C++ statement(s) on output stream out.
-
-void THelix::SavePrimitive(std::ostream &out, Option_t * /*= ""*/)
```

Comment:
What happens if someone calls `SavePrimitives` on a `THelix` after that?  Is it a silent failure or should be add an Error message here instead?

Reply:
User will get message in the produced C macro that SavePrimitive is not implemented for that class

Reply:
By what mechanism?  Isn't `THelix` inheriting from `TPolyLine3D` and thus `TPolyLine3D::SavePrimitives` will be called?

Reply:
Ah yes. Then I will put dummy here.

Reply:
I add alternative implementation. It works somehow, but is not fully cover functionality of THelix.
At least random instance like `THelix *helix = new THelix(5.2, 4.8, 5, 1, 1, 1, 5);` is reproduced.